src/fullnode.rs

```diff
@@ -0,0 +1,108 @@
+use bank::Bank;
+use crdt::{ReplicatedData, TestNode};
+use entry_writer;
+use server::Server;
+use std::fs::File;
+use std::io::{stdin, stdout, BufReader};
+use std::net::SocketAddr;
+use std::sync::atomic::AtomicBool;
+use std::sync::Arc;
+use std::thread::JoinHandle;
+//use std::time::Duration;
+
+pub fn start(
```

Comment:
Can you move this function to `server.rs` and call it `Server::new()`? Congrats.